src/issue.ts

```diff
@@ -278,13 +278,26 @@ function getTypes(i: ApiIssue) {
   return types;
 }
 
-function isPullRequest(i: ApiIssue) {
-  return !!i.PullRequest;
+export function getPriority(p: string): number | undefined {
```

Comment:
Or....

```js
const getPriority = (p: string) => Number(p.toLowerCase().slice(1));
```

No need for any of this I say!

Reply:
ack